includes/utils.php

```diff
@@ -814,6 +814,12 @@ public static function get_super_global_value( $super_global, $key ) {
 		}
 
 		if ( $_FILES === $super_global ) {
+			if ( is_array( $super_global[ $key ]['name'] ) ) {
+				$super_global[ $key ]['name'] = array_map( 'sanitize_file_name', $super_global[ $key ]['name'] );
+
+				return $super_global[ $key ];
+			}
```

Comment:
why do we modify a global value instead of copying it?

Reply:
As it was before